src/controller/meetUpController.js

```diff
@@ -128,11 +136,10 @@ class Controller {
   static async upVote(req, res) {
     try {
       const { rows } = await db.query(queries.selectById('questions', 'id', req.params.questionId));
-      const increase = [(rows[0].votes + 1), rows[0].id];
-      const query2 = 'UPDATE questions SET votes = $1 WHERE id = $2 RETURNING meetup_id,title,body, votes';
-      const resp = await db.query(query2, increase);
-      return res.status(200).json({
-        status: 200,
+      const resp = await db.query(queries.updateVote((rows[0].votes + 1), rows[0].id));
+      const { rowCount } = await db.query(queries.votes(req.params.questionId, req.user.id));
+      return res.status(201).json({
+        status: 201,
```

Comment:
When should status code 201 be returned and do you think it is appropriate to 201 in this situation?

Reply:
Based on further reading i believe it should be 200